data/notyf.min.css

```
@@ -0,0 +1,322 @@
+@-webkit-keyframes notyf-fadeinup {
```

Stylelint (17.3.0)

[error] Unexpected vendor-prefixed at-rule "@-webkit-keyframes" (at-rule-no-vendor-prefix)

(at-rule-no-vendor-prefix)